r10k.yaml.example

```diff
@@ -1,5 +1,79 @@
-:cachedir: '/var/cache/r10k'
-:sources:
-  :plops:
-    remote: 'git@github.com:my-org/org-shared-modules'
-    basedir: '/etc/puppet/environments'
+---
+
+# The cachedir setting controls where cached content, such as mirrored Git
+# repositories, are stored. This location should be persistent, as environments
+# and modules may rely on these files in order to be updated.
+#
+# See https://github.com/puppetlabs/r10k/blob/1.5.1/doc/git/cloning-and-mirroring.mkd
+# for more information on how r10k caches Git repositories.
+#
+# The default value is "~/.r10k"
+cachedir: '/var/cache/r10k'
+
+# The sources setting determines what repositories r10k will use when creating
+# Puppet environments. The structure is a map of source names and a map of the
```

Comment:
I find "a map and a map" a little confusing. Maybe something like "is a map of source names to a map of that source's settings" would be better